zebra-consensus/src/script.rs

```diff
@@ -12,11 +12,181 @@
 //! This is an internal module. Use `verify::BlockVerifier` for blocks and their
 //! transactions, or `mempool::MempoolTransactionVerifier` for mempool transactions.
 
+use std::{pin::Pin, sync::Arc};
+
+use std::future::Future;
+use zebra_chain::{parameters::ConsensusBranchId, transaction::Transaction, transparent};
+
+use crate::BoxError;
+
 /// Internal script verification service.
 ///
 /// After verification, the script future completes. State changes are handled by
 /// `BlockVerifier` or `MempoolTransactionVerifier`.
 ///
 /// `ScriptVerifier` is not yet implemented.
```

Comment:
Is this comment outdated?
```suggestion
/// `ScriptVerifier` is not yet implemented.
```

Reply:
yis

Reply:
Ha oops I forgot to delete the comment in my suggestion.